pypeit/core/flux_calib.py

```diff
@@ -154,24 +154,20 @@ def find_standard_file(ra, dec, toler=20.*units.arcmin, check=False):
                             std_ra=star_coords.ra[_idx].value,
                             std_dec=star_coords.dec[_idx].value)
 
-            if not os.path.isfile(star_file):
```

Comment:
Note, I remove this from here, since it is duplicated on line 133 above. One throws and error, one throws a warning...

Reply:
Nice.